editor/questionbankcontroller.js

```diff
@@ -181,6 +193,37 @@ angular.module('createquestionapp', [])
       $scope.itemsLoading = true;
       $scope.searchQuestions();
       $scope.selectedIndex = undefined;
+      var qsManifest = org.ekstep.pluginframework.pluginManager.getPluginManifest($scope.pluginIdObj.question_set_id);
+      var qsVesrion = qsManifest.ver.split('.')[0];
+      var data = {
+        "request": {
+          "filters": {
+            "objectType": ["Content"],
+            "contentType": ["Plugin"],
+            "targets.id": $scope.pluginIdObj.question_set_id,
+            "targets.ver": {'<=': Number(qsVesrion)},
+            "status": "Live"
+          },
+          "limit": 50,
+          "fields": ['contentType','semanticVersion','appIcon']
+        }
+      };
+      ecEditor.getService('search').search(data, function(err, resp) {
+        var pluginsData = resp.data.result.content;
+        localStorage.setItem("qs-plugins", JSON.stringify(pluginsData));
```

Comment:
Why are you using localStorage? Can't we go with singleton class object store in framework.

Reply:
This should be plugin.js functionality, not the controller functionality. QuestionSet plugin has to load its dependencies.